### PR TITLE
Updating oxi build & enabling alpha optimisations

### PR DESCRIPTION
**Pull request by jakearchibald**
This produces smaller files in some cases, eg: 

grayscale_alpha_16_reduce_alpha_white

I'll add an option to toggle this feature.

### CODE REVIEW

File: codecs/oxipng/Cargo.toml [hunk 1]
```diff
@@ -9,8 +9,8 @@ publish = false
 crate-type = ["cdylib"]
 
 [dependencies]
-oxipng = { version = "2.3.0", default-features = false }
-wasm-bindgen = "0.2.48"
+oxipng = { version = "3.0.0", default-features = false }
+wasm-bindgen = "0.2.64"
```
Comment (RReverser) [inline codecs/oxipng/Cargo.toml:13]: FWIW this is a semver-compatible upgrade, so you don't need to change Cargo.toml.

**Comment by RReverser** [comment]
> I'll add an option to toggle this feature.

To be clear - do you want to add an option here or in a separate PR?

**Comment by jakearchibald** [comment]
If it helps to land this now, I'm happy for it to land without the option. Otherwise I'm not in a rush.

**Comment by RReverser** [comment]
No, it's completely up to you. Just asking to understand whether the PR is ready for review & merge or you want to add more stuff first.

**Comment by jakearchibald** [comment]
Let's look to land it then. I'll add the options in a future PR.

**Review by RReverser — APPROVED** [review]
LGTM.

**Pull request merged**